static/app/components/smartSearchBar/index.tsx

```diff
@@ -560,7 +555,7 @@ class SmartSearchBar extends React.Component<Props, State> {
         childrenIndex !== undefined &&
         searchGroups[groupIndex].children[childrenIndex];
 
-      if (item && !isDefaultDropdownItem(item)) {
```

Comment:
im unsure why we explicitly disabled this before. I dug up the git blame and it goes pretty far back with no clear explanation, not sure if this is dangerous to enable. :/